src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,34 @@
+@import "../utils/mixins.scss";
+
+@import "../utils/variables.scss";
```

Comment:
Move this to main.scss file
```suggestion
```

Reply:
i use variables and mixins only in calendar.scss. So it is useless